idaes/power_generation/carbon_capture/mea_solvent_system/unit_models/tests/phe_validation.py

```diff
@@ -0,0 +1,185 @@
+# Institute for the Design of Advanced Energy Systems Process Systems
+# Engineering Framework (IDAES PSE Framework) Copyright (c) 2018-2019, by the
+# software owners: The Regents of the University of California, through
+# Lawrence Berkeley National Laboratory,  National Technology & Engineering
+# Solutions of Sandia, LLC, Carnegie Mellon University, West Virginia
+# University Research Corporation, et al. All rights reserved.
+#
+# Please see the files COPYRIGHT.txt and LICENSE.txt for full copyright and
+# license information, respectively. Both files are also available online
+# at the URL "https://github.com/IDAES/idaes-pse".
+##############################################################################
+"""
+Comparison of  plate heat exchanger model prediction with NCCC data
+
+Detailed model equations and data can be found in the paper :
+Akula, P., Eslick, J., Bhattacharyya, D. and Miller, D.C., 2019.
+Modelling and Parameter Estimation of a Plate Heat Exchanger
+as Part of a Solvent-Based Post-Combustion CO2 Capture System.
+In Computer Aided Chemical Engineering (Vol. 47, pp. 47-52). Elsevier.
+
+Author: Paul Akula
+"""
+# Import Python libraries and third-party
+import sys
+import os
+import matplotlib.pyplot as plt
+
+# Import Pyomo libraries
+from pyomo.environ import ConcreteModel, SolverFactory, value
+
+# Import IDAES Libraries
+from idaes.core import FlowsheetBlock
+
+# Access the mea_solvent_system dir from the current dir (tests dir)
+sys.path.append(os.path.join(os.path.dirname(__file__), os.pardir, os.pardir))
+
+from unit_models.phe import PHE
+from properties.liquid_prop import LiquidParameterBlock
+
+solver = SolverFactory('ipopt')
+
+m = ConcreteModel()
+m.fs = FlowsheetBlock(default={"dynamic": False})
+# Set up property package
+m.fs.hotside_properties = LiquidParameterBlock()
+m.fs.coldside_properties = LiquidParameterBlock()
+
+# create instance of plate heat exchanger  on flowsheet
+m.fs.hx = PHE(default={
+    "hot_side": {
+        "property_package": m.fs.hotside_properties
+    },
+    "cold_side":
+    {
+        "property_package": m.fs.coldside_properties
+    }})
+
+for t in m.fs.time:
+    # hot fluid
+    m.fs.hx.hot_inlet.flow_mol[t].fix(81.6322)
+    m.fs.hx.hot_inlet.temperature[t].fix(392.23)
+    m.fs.hx.hot_inlet.pressure[t].fix(182840)
+    m.fs.hx.hot_inlet.mole_frac_comp[t, "CO2"].fix(0.01585)
+    m.fs.hx.hot_inlet.mole_frac_comp[t, "H2O"].fix(0.87457)
+    m.fs.hx.hot_inlet.mole_frac_comp[t, "MEA"].fix(0.10958)
+
+    # cold fluid
+    m.fs.hx.cold_inlet.flow_mol[t].fix(84.7399)
+    m.fs.hx.cold_inlet.temperature[t].fix(326.36)
+    m.fs.hx.cold_inlet.pressure[t].fix(182840)
+    m.fs.hx.cold_inlet.mole_frac_comp[t, "CO2"].fix(0.04142)
+    m.fs.hx.cold_inlet.mole_frac_comp[t, "H2O"].fix(0.85078)
+    m.fs.hx.cold_inlet.mole_frac_comp[t, "MEA"].fix(0.10780)
+
+m.fs.hx.initialize(outlvl=0)
+
+# model results
+PHE_THOUT = []
```

Comment:
In light of my other comments, you do not need these to store the results, as you should be checking the values at each operating condition as you go.